scripts/import/import_all_sv_data.pl

```diff
@@ -1530,8 +1532,19 @@ sub parse_9th_col {
     
     # Phenotype
     if ($key eq 'phenotype') {
+      # Some phenotypes contain a comma in the description
+      # Replace the comma before splitting by comma
```

Comment:
The comment `Replace the comma before splitting by comma` could be clearer. Can you add a note that the input is a string of multiple phenotypes separated by comma?

Reply:
I'll change it to make it more clear.